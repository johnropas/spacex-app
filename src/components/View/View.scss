@import "../../styles/variables";
@import "../../styles/global";
@import "../../styles/functions";

.sub-title {
  font-size: $fontSizeMedium;
  font-weight: $regular;
}
.main {
  // padding: calcRem(48px);
  @include media-breakpoint-down(sm) {
    padding: 0;
  }
  flex: 5;

  .title {
    font-size: $fontSizeMedium;
    font-weight: $bold;
  }
}
