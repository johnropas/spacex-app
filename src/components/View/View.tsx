import React from 'react';
import './View.scss';
import Header from '../Header';
import Footer from '../Footer';
import MobileMenuBottom from '../MobileBottomMenu';

const View = (props: any) => {
  return (
    <div>
      <div className="">
        <div className="">
          <div className="no-padding">
            <Header />
            <div className="main">{props.children}</div>
            <div className="footer-mobile-nav">
              <MobileMenuBottom />
            </div>
            <Footer />
          </div>
        </div>
      </div>
    </div>
  );
};

export default View;
