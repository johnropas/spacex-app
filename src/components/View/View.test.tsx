import React from 'react';
import renderer from 'react-test-renderer';
import View from './View';

it('Instantiates View Component', () => {
  const component = renderer.create(<View />);
  expect(component).toBeTruthy();
});

it('Renders View Component', () => {
  const component = renderer.create(<View />);
  expect(component).toMatchSnapshot();
});
