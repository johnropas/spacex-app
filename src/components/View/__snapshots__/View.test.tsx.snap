// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders View Component 1`] = `
<div>
  <div
    className=""
  >
    <div
      className=""
    >
      <div
        className="no-padding"
      >
        <div
          className="main"
        />
      </div>
    </div>
  </div>
</div>
`;
